cache_images/fedora_packaging.sh

```diff
@@ -194,6 +194,12 @@ DOWNLOAD_PACKAGES=(\
 msg "Installing general build/test dependencies"
 bigto $SUDO dnf install -y "${INSTALL_PACKAGES[@]}"
 
+timebomb 20240214 "Gaaaaaaaaah! Kludge to emergency-get netavark 1.10.3"
+arch=$(uname -m)
+fedoraversion=$(awk -F= '$1 == "VERSION_ID" { print $2 }' </etc/os-release)
```

Comment:
This is available from the automation-library as `$OS_RELEASE_VER`

Reply:
Thank you! I'll try to remember for next time. Right now, this is timebombed, so I see no reason to repush.